src/user-guide/running-workflows/reflow.rst

```diff
@@ -1,86 +1,227 @@
+that point. Downstream tasks belong to both flows.
+
+Flow merging in ``n=0`` means flows are not completely independent. One flow
+might not be able to entirely overtake another because one or more of its tasks
+might merge in ``n=0``. Merging is necessary while task IDs - and associated
+log directory paths etc. - do not incorporate flow numbers, because task IDs
+must be unique in the :term:`active task pool`.
+
+Merging with Incomplete tasks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:term:`Incomplete<incomplete>` tasks are retained in the active window in
+expectation of retriggering to complete :term:`expected outputs<expected
+output>` and continue their flow.
+
+If another flow encounters an incomplete task (i.e. if another instance of the
+same task collides with it in the ``n=0`` :term:`active window`) the task will
+run again and carry both flow numbers forward if it successfully completes its
+expected outputs.
 
-If a task from one flow catches up with an active sibling from another
-(i.e. an active task with the same name and :term:`cycle point`, but a
-different flow number) they will merge and carry both flow numbers
-forward. Downstream tasks can be considered to belong to either flow.
-Any number of flows can merge like this.
 
 Stopping Flows
 --------------
 
-By default, ``cylc stop`` halts the entire workflow and shuts the scheduler down.
+By default, ``cylc stop`` halts the workflow and shuts the scheduler down.
+
+It can also stop specific flows: ``cylc stop --flow=N`` removes the flow number
+``N`` from tasks in the :term:`active task pool`. Tasks that have no flow
+numbers left as a result do not spawn children at all. If there are no active
+flows left, the scheduler shuts down.
+
+.. TODO update this section post https://github.com/cylc/cylc-flow/issues/4741
+
+
+.. _flow-trigger-use-cases:
+
+Some Use Cases
+--------------
+
+Running Tasks Ahead of Time
+   To run a task within the existing flow(s) even though its prerequisites are
+   not satisfied yet, just trigger it. Use ``--wait`` if you don't want the new
+   flow front to continue immediately. Triggered task(s) will not re-run when
+   the main front catches up.
+
+Regenerating Products Behind a Flow
+   To re-run a sub-graph (e.g. because the original run was affected by a
+   corrupt file), just trigger the task(s) at the top of the sub-graph with
+   ``--flow=new``.
+
+   You may need to manually stop the new flow once its job is done, to avoid
+   re-running more than you want to, if the new flow leads into the main
+   trunk of the graph.
+
+Rewinding a Workflow
+   To rewind the workflow to an earlier point, perhaps to regenerate data and/or 
+   allow the workflow to evolve a new path into the future, trigger a new
+   flow at the right place and then stop the original flow. (Alternatively,
+   stop the scheduler, install a new instance of the workflow, and play it
+   from the desired place in the graph).
```

Comment:
Installing a new instance isn't the same because you lose the contents of the `share` directory.
I'd suggest removing the bit  in brackets.